specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/examples/Clusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "api-version": "2016-09-01-preview",
+    "subscriptionId": "{subscription-id}",
```

Comment:
[nit] find+replace {subscription-id} in these examples with something that resembles real data (random guid)